src/components/Calendar/index.js

```diff
@@ -43,9 +44,13 @@ class Calendar extends PureComponent {
     this.estimateMonthSize = this.estimateMonthSize.bind(this);
     this.handleScroll = this.handleScroll.bind(this);
     this.dateOptions = { locale: props.locale };
+    if (this.props.weekStartsOn !== undefined)
```

Comment:
You can use `if(!!props. weekStartsOn)`

Reply:
`!!0` returns false, while `0 !== undefined` returns true. Current one is OK since we have to enter into if block when `weekStartsOn=0` (0 means Monday)